weather.py

```diff
@@ -108,6 +108,14 @@ def raining_effect(affected_worms: List[Worm], affected_food: List[Food]) -> Non
             for item in affected_food:
                 item.nutritional_value -= 0.5
 
+    def get_side(self) -> int:

```

Comment:
https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters

You can try decorators for setter/getter.

For example look how it's done here: https://github.com/jtpereyda/boofuzz/blob/master/boofuzz/blocks/request.py#L67